lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -253,7 +253,7 @@ def _apt(
             subprocess.run(_cmd, capture_output=True, check=True, env=env)
         except CalledProcessError as e:
             raise PackageError(
-                "Could not {} package(s) [{}]: {}".format(command, [*package_names], e.output)
+                "Could not {} package(s) [{}]: {}".format(command, [*package_names], e.stderr)
```

Comment:
Regarding bytes vs str, if we provide the `text=True` arg to `subprocess.run`, it'll be a str.

Reply:
Yup. Will do.